app/api/endpoints/lossClaim/list/v3/ListLossClaimsController.scala

```diff
@@ -50,6 +52,7 @@ class ListLossClaimsController @Inject()(val authService: EnrolmentsAuthService,
            businessId: Option[String],
            typeOfClaim: Option[String]): Action[AnyContent] =
     authorisedAction(nino).async { implicit request =>
+      implicit val correlationId: String = idGenerator.getCorrelationId
```

Comment:
Can this be spaced out a bit, it's all a bit bunched together

Reply:
Fixed